### DISCUSSION TITLE
Homepage problems

### DISCUSSION THREAD

**Discussion by ocularvr** [mozilla/hubs#2991] [General Questions & Discussion] [answered]
Hi, the home page is not displaying any of the texts provided in translation except for "App Tagline". 
Please advise.
![expected](https://user-images.githubusercontent.com/69241952/92589872-87c66400-f29b-11ea-9613-9b12cba81059.jpg)
![result](https://user-images.githubusercontent.com/69241952/92589873-885efa80-f29b-11ea-8257-f71c4ef34039.jpg)

**Comment by robinkwilson** [comment, marked as answer]
~~Tested it myself, you're right here's the issue to follow:~~

~~https://github.com/mozilla/hubs-cloud/issues/105~~

~~Thanks for flagging it! I'll bubble this up to the team.~~

**Reply by robinkwilson** [reply]
My mistake, these are the expected behavior on a vanilla setup. Do you have anything custom that would interfere with the text rendering?

**Reply by robinkwilson** [reply]
![image](https://user-images.githubusercontent.com/29560735/92634565-56449d00-f289-11ea-8768-26eec42a28a0.png)

![image](https://user-images.githubusercontent.com/29560735/92634578-5a70ba80-f289-11ea-9828-c1482f6017e8.png)
